apps/web/src/pages/Auth/CreateAccountPage/CreateAccountRedirect.tsx

```diff
@@ -33,7 +35,7 @@ const CreateAccountRedirect = () => {
      *  - User has no email associated with account
      *  - User is not trying to go to the welcome page directly already
      */
-    if (loggedIn && authorizationLoaded && empty(email) && isToCreateAccount) {
+    if (loggedIn && !fetching && empty(email) && isToCreateAccount) {
```

Comment:
I just realized that the `isLoaded` value from `useAuthorization` is actually `const isLoaded = !fetching && !stale`, so it may be a good idea to use both fetching and stale here as well.

Reply:
Sure, we can do that. 
I added in a reference to `stale` here in https://github.com/GCTC-NTGC/gc-digital-talent/pull/8297/commits/4dbe55f9f74fe94132b00e8fad114f08beea8f23